Package.swift

```diff
@@ -104,8 +104,8 @@ var dependencies: [Package.Dependency] {
     } else {
         [
             .package(
-                url: "https://github.com/apple/swift-foundation-icu",
-                from: "0.0.9"),
+                url: "https://github.com/apple/swift-foundation-icu.git",
```

Comment:
Is the addition of `.git` and the change from `from` to `exact` intentional?

Reply:
Ahh the exact part is... since it seems every ICU upgrade are somewhat breaking so we should intentionally upgrade.